news/2023/2023-07-06-the-roundtable-ii.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "The Roundtable II"
+date: 2023-07-06 19:00:00 +0000
+---
+
+16 of osu!'s fiercest tournament players battle in this year's edition of **The Roundtable**, a major in-person invitational tournament hosted by [BTMC](https://osu.ppy.sh/users/3171691)!
+
+![](/wiki/shared/news/2023-07-06-the-roundtable-ii/banner.jpg)
+
+**The Roundtable II** takes the concept behind [last year's prototype tournament](/wiki/Tournaments/The_Roundtable) and raises the bar tenfold.
+
+This **3-day** event from **July 7–9** features **2 major tournaments** and a bundle of side events, all streamed from [BTMC](https://osu.ppy.sh/users/3171691)'s [Twitch channel](https://twitch.tv/BTMC)!
+
+## Schedule
+
+### Tournaments
+
+| Event | Day | Timestamp (UTC) |
+| --: | :-- | :-- |
+| [**16-player battle royale**](#16-player-battle-royale) | Day 1 | 2023-07-07 (18:00) |
+| [**1v1 double-elimination tournament**](#1v1-double-elimination-tournament) | Day 2 | 2023-07-08 (19:00) |
+| [**1v1 double-elimination tournament**](#1v1-double-elimination-tournament) | Day 3 | 2023-07-09 (17:30) |
+
+#### 16-player battle royale
+
+Before diving into the main event, players will be warming up their competitive drives with a **16-player battle royale**!
+
+The LAN setup has 8 computers, so players will be split into two groups of 8. Each group will play **7 maps** in a multiplayer lobby, and whoever scores the lowest on each map will be eliminated.
+
+Once both groups have been narrowed down to a single player, the finalists will face head to head in a **best-of-13 competition!**
+
+#### 1v1 double-elimination tournament
+
+Everything is on the line for **The Roundtable II**'s main event: a **double-elimination 1v1 tournament**!
+
+This tournament will last two days:
+
+| Day | Brackets |
+| --: | :-- |
+| Day 2 | Round of 16, Qualifiers |
+| Day 3 | Semifinals, Finals, Grand Finals |
+
+Witness osu!'s top players duke it out *in person* for unending glory. (And cool prizes, of course.)
+
+### Side events
+
+| Event | Day | Timestamp |
+| --: | :-- | :-- |
+| **osu! Scavenger Hunt** | Day 1 | 2023-07-07 (20:00) |
+| **Are You Smarter Than an osu! Player?** | Day 1 | 2023-07-07 (23:00) |
+| **Blitz Jenga** | Day 2 | 2023-07-08 (18:00) |
+| **Reaction Time Buzzer Contest** | Day 2 | 2023-07-09 (02:30) |
+| **Hot Ones Hot Wings Challenge** | Day 2 | 2023-07-09 (04:00) |
```

Comment:
```suggestion
| **Reaction Time Buzzer Contest** | Day 3 | 2023-07-09 (02:30) |
| **Hot Ones Hot Wings Challenge** | Day 3 | 2023-07-09 (04:00) |
```

Reply:
@pishifat are these supposed to be on the same day due to time zones? if so it'd look less weird if the timestamps were in utc-07 for la time

Reply:
I think utc0 is good for universal understandability. day 2 refers to the second day of the stream